apollo-router/src/plugins/telemetry/metrics/prometheus.rs

```diff
@@ -50,7 +49,7 @@ fn prometheus_default_path() -> String {
 impl Default for Config {
     fn default() -> Self {
         Self {
-            enabled: true,
+            enabled: false,
```

Comment:
It concerns me that this wasn't always `false`...

Reply:
Agree, the root of our issues is this: https://github.com/serde-rs/serde/issues/2622. Depending on where the defaulting happens the value may have been true or false.